vtex/faststore
# Feat: Set up Docusaurus + Adds Radio Tokens doc

## What's the purpose of this pull request?

- [x] Set docusaurus config to work with .scss and import global tokens
- [x] Adds Tokens Component (`TokenTable`, `TokenRow`, `TokenDivider`)
- [x] Updates Radio doc page

## How it works?

Adds design tokens and updates Radio documentation template on '/docs'

## How to test it?

See the [preview](https://faststore-git-chore-radio-doc-set-up-faststore.vercel.app/reference/ui/atoms/Radio) and check if any important information is missing for using this component.

## References
